cuckoo/web/utils.py

```diff
@@ -132,3 +132,14 @@ def normalize_task(task):
             os.path.basename(task["target"])
         )
     return task
+
+def get_client_ip(request):
+    try:
+        x_forwarded_for = request.META.get('HTTP_X_FORWARDED_FOR')
```

Comment:
`X-Forwarded-For` can be spoofed easily.

ref: https://portswigger.net/kb/issues/00400110_spoofable-client-ip-address

Reply:
We agree, this part was edited out and merged manually after that, without that header. :slightly_smiling_face:

https://github.com/cuckoosandbox/cuckoo/blob/master/cuckoo/web/web/middle.py#L36